eng/pipelines/libraries/enterprise/linux.yml

```diff
@@ -11,6 +11,7 @@ pr:
 
   paths:
     include:
+      - eng/pipelines/libraries/enterprise/*
```

Comment:
Why do we now need an extra path?

Reply:
Without adding this, my pipeline doesn't get invoked automatically with PRs if this pipeline file changes itself.

@safern [mentioned](https://github.com/dotnet/runtime/pull/463#discussion_r354638128) that I should fix this

Reply:
I see, makes sense.